src/models.py

```diff
@@ -50,3 +50,17 @@ class Topic(Base):  # pylint: disable=too-few-public-methods
     description = Column(String(123), nullable=False)
     title = Column(String(30), nullable=False)
     author: User = relationship("User", uselist=False)
+    posts: list["Post"] = relationship("Post", uselist=True)
```

Comment:
> There is a problem with this solution: I couldn't find a way to explicitly order posts by creation date. They do render the way we want them to, but "it just works", without my interference. I don't see a way to change that without changing routes.topic view function.

Try this:
```python
posts: list[Post] = relationship("Post", order_by="desc(Post.created_at)")
```

But worth to mention that changing `routes.topic` function is absolutely acceptable solution here as well.

Reply:
And let's get rid of quotes inside type annotation, to be consistent with other annotations.

Reply:
Also there is no sense to add `uselist=True` since this is the default value.

Reply:
> And let's get rid of quotes inside type annotation, to be consistent with other annotations.

Post model is defined after Topic, therefore linters as well as VScode itself identify it as an error.

> Also there is no sense to add `uselist=True` since this is the default value.

Same here, even though it is a default value, if I don't specify it in the code mypy raises an error.

Reply:
I've removed `uselist=True` and mypy gives me no errors.

To resolve a problem with quotes you should add _magic_ import at the top of the module, like that:
```python
"""forum123's database models module."""

from __future__ import annotations

import hashlib

from sqlalchemy import Column, DateTime, ForeignKey, func, Integer, String
from sqlalchemy.orm import relationship
```